tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -247,6 +247,83 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
+	printk("enabling...");
```

Comment:
For all tests: this only tests that the HCI commands are working as intended. Is it possible to add tests for the actual functionality behind the HCI commands?

Reply:
The old tests use HCI commands. The new tests use link layer calls. I don't think we can easily get more detailed than that.

Reply:
My point is that you've tested that the HCI interface is working, but how do we know that the adv aux set really was removed (i.e. that the function ll_adv_aux_set_remove works as intended)

Reply:
I guess we can make a second call to ll_adv_aux_set_remove. If the set is removed (is_created=0), we should expect the call to fail with BT_HCI_ERR_UNKNOWN_ADV_IDENTIFIER.

Reply:
Actually, we alreadyhave this test - "Removing adv aux set that's not created".